Adapter.H

```diff
@@ -106,7 +105,7 @@ private:
     std::vector<Interface*> interfaces_;
 
     //- preCICE solver interface
-    precice::SolverInterface* precice_ = NULL;
+    precice::Participant* precice_ = NULL;
```

Comment:
Since we are changing this, let's directly make it nullptr:

```suggestion
    precice::Participant* precice_ = nullptr;
```